examples/monolithic_build/config_a.h

```diff
@@ -43,15 +43,16 @@
  * Description: The macros to use to namespace global symbols
  *              from mlkem/.
  *****************************************************************************/
-#define MLKEM_NAMESPACE(sym) MLKEM_DEFAULT_NAMESPACE(sym)
+#define CONCAT(a, b) a##b
+#define MLKEM_NAMESPACE(sym) CONCAT(namespace_a_, sym)
```

Comment:
why not just use the default namespace? it includes the parameter set so there should not be any clashes?

Reply:
Actually in this case there would still be clashes for FIPS202, which does not include the level.